runtime/pretty/print.go

```diff
@@ -315,6 +315,10 @@ func (p ErrorPrettyPrinter) writeCodeExcerpts(
 				indicatorLength = maxLineLength
 			}
 
+			if indicatorLength >= len(line) {
+				indicatorLength = len(line)
+			}
```

Comment:
Likely related to #2021. The pretty printer should not gracefully handle invalid input, we should fix the underlying problem resulting in invalid input

```suggestion
```